light_prop/utils.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import numpy as np
+from PIL import Image
+
+
+def image_to_amplitude(image_path):
+    array = ndarray_to_image(image_path)
+    return normalize_array(array, 1)
+
+
+def image_to_phase(image_path):
+    array = ndarray_to_image(image_path)
+    return normalize_array(array, 2 * np.pi)
+
+
+def amplitude_to_image(array: np.array, file_path="output", file_extension="bmp"):
```

Comment:
Why are you not producing/consuming our data type `LightField`?

Reply:
Lightfield needs to check some stuff like "has amp the same size as phase", "is it corresponding to current data" and I was prototyping it to have a single responsibility. We can write something on top of of it